keyboards/electronlab/klor/keyboard.json

```diff
@@ -0,0 +1,382 @@
+            {"matrix": [0, 1], "x": 13, "y": 17, "flags": 4},
+            {"matrix": [0, 2], "x": 29, "y": 7, "flags": 4},
+            {"matrix": [0, 3], "x": 46, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 60, "y": 6, "flags": 4},
+            {"matrix": [0, 5], "x": 75, "y": 8, "flags": 4},
+            {"matrix": [4, 5], "x": 149, "y": 8, "flags": 4},
+            {"matrix": [4, 4], "x": 163, "y": 6, "flags": 4},
+            {"matrix": [4, 3], "x": 178, "y": 0, "flags": 4},
+            {"matrix": [4, 2], "x": 195, "y": 7, "flags": 4},
+            {"matrix": [4, 1], "x": 206, "y": 42, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 28, "flags": 1},
+            {"matrix": [1, 1], "x": 15, "y": 30, "flags": 4},
+            {"matrix": [1, 2], "x": 30, "y": 20, "flags": 4},
+            {"matrix": [1, 3], "x": 46, "y": 12, "flags": 4},
+            {"matrix": [1, 4], "x": 60, "y": 19, "flags": 4},
+            {"matrix": [1, 5], "x": 75, "y": 21, "flags": 4},
+            {"matrix": [5, 5], "x": 149, "y": 21, "flags": 4},
+            {"matrix": [5, 4], "x": 163, "y": 19, "flags": 4},
+            {"matrix": [5, 3], "x": 178, "y": 12, "flags": 4},
+            {"matrix": [5, 2], "x": 194, "y": 20, "flags": 4},
+            {"matrix": [5, 1], "x": 209, "y": 30, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 28, "flags": 1},
+            {"matrix": [2, 0], "x": 3, "y": 41, "flags": 1},
+            {"matrix": [2, 1], "x": 17, "y": 42, "flags": 4},
+            {"matrix": [2, 2], "x": 31, "y": 33, "flags": 4},
+            {"matrix": [2, 3], "x": 46, "y": 25, "flags": 4},
+            {"matrix": [2, 4], "x": 60, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 75, "y": 34, "flags": 4},
+            {"matrix": [6, 5], "x": 149, "y": 34, "flags": 4},
+            {"matrix": [6, 4], "x": 163, "y": 32, "flags": 4},
+            {"matrix": [6, 3], "x": 178, "y": 25, "flags": 4},
+            {"matrix": [6, 2], "x": 193, "y": 33, "flags": 4},
+            {"matrix": [6, 1], "x": 211, "y": 17, "flags": 4},
+            {"matrix": [6, 0], "x": 221, "y": 41, "flags": 1},
+            {"matrix": [3, 1], "x": 58, "y": 48, "flags": 1},
+            {"matrix": [3, 2], "x": 74, "y": 49, "flags": 1},
+            {"matrix": [3, 3], "x": 89, "y": 54, "flags": 1},
+            {"matrix": [3, 4], "x": 102, "y": 63, "flags": 1},
+            {"matrix": [7, 4], "x": 122, "y": 63, "flags": 1},
+            {"matrix": [7, 3], "x": 135, "y": 54, "flags": 1},
+            {"matrix": [7, 2], "x": 150, "y": 49, "flags": 1},
+            {"matrix": [7, 1], "x": 166, "y": 48, "flags": 1}
+        ],
+        "split_count": [21, 21],
+        "led_flush_limit": 16,
+        "led_process_limit": 10,
+        "max_brightness": 255,
+        "react_on_keyup": false,
+        "sat_steps": 16,
+        "sleep": true,
+        "speed_steps": 16,
+        "timeout": 120000,
+        "val_steps": 16,
+        "hue_steps": 3
```

Comment:
Agreed. Change is done and committed.
But it would be nice to expose these settings even with default as some people have a hard time finding how to tune them.